models/wholeBody_ct_segmentation/configs/evaluate.json

```diff
@@ -7,21 +7,6 @@
                 "keys": "pred",
                 "softmax": true
             },
-            {
-                "_target_": "Invertd",
```

Comment:
I thought `Invertd` is necessary for evaluation to get more accurate right?

Thanks.

Reply:
Maybe we could just remove `to_onehot` here to be GPU mem friendly, just like what I do in this [PR](https://github.com/Project-MONAI/model-zoo/pull/397/files#diff-13d7daf2368303783ff1fb16c647118e08e62600c7d28f0a0102f62626ea1351).

But also need to update `MeanDice` in the train.json to be:
```
"val_mean_dice": {
                "_target_": "MeanDice",
                "include_background": false,
                "output_transform": "$monai.handlers.from_engine(['pred', 'label'])",
                "num_classes": 105    # add this line
}
```

Reply:
Hi @tangy5 , could you please address this question. Please also ensure the changes exactly align with eval metrics (metedata.json, README) and GPU memory (README).

Reply:
@KumoLiu @yiheng-wang-nv , thanks.

After discussing with the QA team, we will not make this evaluation change. We will keep everything as original and keep evaluation pipeline consistent. Instead, we will make more detailed documentions and isntruct users how to run low-res and high-res evaluation in a friendly way, if users doesn't has a high mem GPU, they can do low-res, or they can directly run inference instead of evaluation. 

Thanks.